### ISSUE TITLE
Accessibility issue: Aria input fields must have an accessible name on /notebook-ci-testing/notebooks/ACS/acs_focus_diverse_epsfs/acs_focus_diverse_epsfs.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#254]
## What
An accessibility scan flagged an element on https://spacetelescope.github.io/notebook-ci-testing/notebooks/ACS/acs_focus_diverse_epsfs/acs_focus_diverse_epsfs.html because aria input fields must have an accessible name. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/aria-input-field-name?application=playwright.

The following element needs attention:

- `<div class="noUi-handle noUi-handle-lower" data-handle="0" tabindex="0" role="slider" aria-orientation="horizontal" aria-valuemin="0.0" aria-valuemax="89.0" aria-valuenow="44.0" aria-valuetext="44"><div class="noUi-touch-area"></div></div>` (selector: `.noUi-handle`)

To fix this, ensure every aria input field has an accessible name.

Specifically:

Fix any of the following:
-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Element has no title attribute

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
